transaction/models.py

```diff
@@ -17,6 +18,15 @@ class TransactionStaticData(models.Model):
                             help_text='The type of transaction (income or expense)')
     description = models.TextField(help_text='A short description of what the transaction is for')
     notes = models.TextField(blank=True, help_text='A longer description and details about the transaction')
+    categories = models.ForeignKey(
```

Comment:
Why is the name plural when you are using a ForeignKey, which only points to one object?

Reply:
I intended it to point to more than one.  I suppose `ManyToManyField` is what I need then?

Reply:
If I recall correctly, Anders said he only needed one category per transaction. But we could of course still use a many-to-many field just in case. @FredrikAugust @NicolajNiels1 @KaizerTilt  What do you think?

Reply:
My impression when discussing categories with Anders was that a transaction could only have one category.